x/evm/types/params.go

```diff
@@ -40,19 +37,15 @@ var (
 	AvailableExtraEIPs = []int64{1344, 1884, 2200, 2929, 3198, 3529}
 )
 
-// ParamKeyTable returns the parameter key table.
-func ParamKeyTable() paramtypes.KeyTable {
-	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
-}
-
 // NewParams creates a new Params instance
-func NewParams(evmDenom string, enableCreate, enableCall bool, config ChainConfig, extraEIPs ...int64) Params {
+func NewParams(evmDenom string, allowUnprotectedTxs, enableCreate, enableCall bool, config ChainConfig, extraEIPs ExtraEIPs) Params {
```

Comment:
Was this a bug before? Looks like it wasn't included in the params.

Reply:
I'm not entirely sure I assumed it was a bug since it had a key and was part of the params. @fedekunze can maybe weigh in on this?

Reply:
Actually it makes sense that we don't expose this to governance change?